pkg/libb2/include/libb2_config.h

```diff
@@ -0,0 +1,24 @@
+#ifndef LIBB2_CONFIG_H
+#define LIBB2_CONFIG_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Add no suffix to the functions */
+#define SUFFIX
+
+/* Test for a little-endian machine */
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+#define NATIVE_LITTLE_ENDIAN
+#endif
+
+/* CPUs that don't support unaligned access, taken from GStreamer */
+#if defined(__alpha__) || defined(__arc__) || defined(__arm__) || defined(__aarch64__) || defined(__bfin) || defined(__hppa__) || defined(__nios2__) || defined(__MICROBLAZE__) || defined(__mips__) || defined(__or1k__) || defined(__sh__) || defined(__SH4__) || defined(__sparc__) || defined(__sparc) || defined(__ia64__) || defined(_M_ALPHA) || defined(_M_ARM) || defined(_M_ARM64) || defined(_M_IA64) || defined(__xtensa__) || defined(__e2k__) || defined(__riscv)
+#define HAVE_ALIGNED_ACCESS_REQUIRED
```

Comment:
@benpicco Does this cover all RIOT architectures?

Reply:
@benpicco Please insert some line breaks in the condition to fit the style guide requirement of having usually 80 characters per line and not more than 100 characters per line.

Reply:
Oh you are right. The list looked complrehensive enough to cover all possible future ports ;) But it is indeed unlikely that Linux/GStreamer would ever run on MSP430 or Atmega but I see they would also not support unaligned memory access.

Reply:
I think this belongs to `cpu/` where CPUs that support it can define `CPU_HAVE_UNALIGNED_ACCESS`, but this is for another PR.

(thinking about it, the only architecture we support that actually can do unaligned load would be `native` on x86. Well, also ARM, but that is with a little help from the kernel.